config/crd/patches/webhook_in_clusterbootstraps.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
+# CRD conversion requires k8s 1.13 or later.
```

Comment:
same for webhook_in_* files , we need to decide if we include them for all controllers or remove them.

Reply:
Removed